src/components/popover.svelte

```diff
@@ -0,0 +1,20 @@
+<script lang="ts">
+  import clsx from 'clsx';
+</script>
+
+<div class="group relative inline-block">
+  <div role="tooltip" class="cursor-default">
+    <slot name="target" />
+  </div>
+  {#if $$slots['popover']}
+    <div
+      class={clsx(
+        'absolute z-10 top-0 left-1/2 md:-top-[90%] md:-left-[97%] w-72 p-4',
+        'invisible group-hover:visible group-hover:opacity-100 transition-opacity opacity-0 bg-background',
```

Comment:
I think also having `focus-within` triggering it would benefit keyboard navigation

Reply:
Also I'm unable to trigger it with a touch (mobile) device.